package.json

```diff
@@ -14,6 +14,11 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@vx/gradient": "0.0.136",
+    "@vx/group": "0.0.136",
+    "@vx/mock-data": "0.0.136",
```

Comment:
As this is mock data (not to go into production), can this package be installed with `npm install –save-dev`?